src/app/app-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+
+import { RouterModule, Routes } from '@angular/router';
+import { AssesReportsComponent } from './asses-reports/asses-reports.component';
+import { AdminPanelComponent } from './admin-panel/admin-panel.component';
+import { HomePageComponent } from './home-page/home-page.component';
+
+import { GraphComponent } from './graph/graph.component';
+import { AuthGuard } from './auth.guard';
+
+export const routes: Routes = [
+  {
+    path: '',
+    component: AssesReportsComponent,
+    canActivate: [AuthGuard],
+  },
+  {
+    path: 'home',
+    component: HomePageComponent,
+  },
```

Comment:
make HomePage default component (for `path: ''` or `if path: '', redirect to path: 'home'`
and implement form within this component instead of opening modal window